src/lib/ng-package/package.ts

```diff
@@ -78,12 +78,13 @@ export class NgPackage {
     const alwaysInclude = ['tslib'];
 
     // remove in the future, after deprecation phase
-    const deprecatedPropList = this.whitelistedNonPeerDependencies;
+    let deprecatedPropList = this.whitelistedNonPeerDependencies;
```

Comment:
Can be improoved: 

`const deprecatedPropList = this.whitelistedNonPeerDependencies ?? [];`